internal/generator/vector/conf_test.go

```diff
@@ -118,6 +118,9 @@ type = "file"
 ignore_older_secs = 600
 include = ["/var/log/oauth-apiserver.audit.log"]
 
+[sources.internal_metrics]
+type = "internal_metrics"
```

Comment:
We cannot add namespace in this vector version. In latest version we can set the metric namespace here.